Library/Formula/innotop.rb

```diff
@@ -23,7 +21,7 @@ def install
     end
 
     system "perl", "Makefile.PL", "PREFIX=#{prefix}"
-    system "make install"
+    system "make", "install"
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!